From 0af53b212bf26b0c94855dab116a483c42ce19e1 Mon Sep 17 00:00:00 2001
Message-ID: <0af53b212bf26b0c94855dab116a483c42ce19e1.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 15 Jan 2024 20:15:54 +0000
Subject: [PATCH 16/19] gnu: python-stsynphot: Fix build.

* gnu/packages/astronomy.scm (python-stsynphot): Fix build.
[native-inputs]: Add python-semantic-version.

Change-Id: I72b086c6a0461293c2191e39e25b3c803279bf60
---
 gnu/packages/astronomy.scm | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 5adba72439..dd7473df49 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3036,8 +3036,11 @@ (define-public python-stsynphot
                              python-numpy
                              python-scipy
                              python-synphot))
-    (native-inputs (list python-pytest python-pytest-astropy
-                         python-pytest-astropy-header python-setuptools-scm))
+    (native-inputs (list python-pytest
+                         python-pytest-astropy
+                         python-pytest-astropy-header
+                         python-semantic-version
+                         python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/stsynphot_refactor")
     (synopsis "Synthetic photometry using Astropy for HST and JWST")
     (description
-- 
2.41.0

